core/src/main/resources/lib/hudson/artifactList_de.properties

```diff
@@ -21,3 +21,6 @@
 # THE SOFTWARE.
 
 View=anzeigen
+view=anzeigen
```

Comment:
@imod Right, grammar fail. I tried saying that having two similar entries is actually correct.